### PR TITLE
show prediction even on vi command mode

### PR DESCRIPTION
**Pull request by devank4000**

### CODE REVIEW

File: lineedit/editing.c [hunk 1]
```diff
@@ -2572,11 +2572,14 @@ void clear_prediction(void)
 void update_buffer_with_prediction(void)
 {
     clear_prediction();
+    int effective_index;
 
     if (!shopt_le_predictempty && active_length() == 0)
         return;
 
-    if (le_main_index < active_length())
+    effective_index = LE_CURRENT_MODE == LE_MODE_VI_COMMAND ?
```
Comment (magicant) [inline lineedit/editing.c]: `le_main_index` is not an `int`.
Also, the variable type declaration should be in the same statement as the first assignment.

```suggestion
    size_t effective_index = LE_CURRENT_MODE == LE_MODE_VI_COMMAND ?
```

File: lineedit/editing.c [hunk 2]
```diff
@@ -2572,11 +2572,14 @@ void clear_prediction(void)
 void update_buffer_with_prediction(void)
 {
     clear_prediction();
+    int effective_index;
```
Comment (magicant) [inline lineedit/editing.c]:
```suggestion
```

File: lineedit/editing.c [hunk 3]
```diff
@@ -2586,7 +2589,6 @@ void update_buffer_with_prediction(void)
     wb_catfree(&le_main_buffer, suffix);
 }
 
-
```
Comment (magicant) [inline lineedit/editing.c]: A section header should be preceded by two empty lines.

**Comment by devank4000** [comment]
Thanks for the feedback. I've made the changes in the latest commit.

**Comment by magicant** [comment]
thanks

**Pull request merged**